### ISSUE TITLE
Sort-date attribute is always 0

### ISSUE THREAD

**Issue by CervDotBe** [pippyn/Home-Assistant-Sensor-Afvalbeheer#67]
Hello, 

I want to make use of the sort-date attribute as described in this resolved issue : https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer/issues/33

However, the sort-date attribute stays 0 whatever I try.

![image](https://user-images.githubusercontent.com/29871395/80471854-449ad280-8944-11ea-9627-ba94b61ddbbf.png)

My config:
```
  - platform: afvalbeheer
    wastecollector: Ophaalkalender
    resources:
      - restafval
      - gft
      - papier
      - pmd
      - glas
      - textiel
    postcode: ZZZZ
    streetname: ZZZZ
    streetnumber: Z
    dateonly: 1
    dateformat: '%d/%m/%Y'
    dateobject: 1
```

Thanks in advance for your help.

**Comment by pippyn** [comment]
Thank you for reporting this bug. Will fix asap

**Comment by pippyn** [comment]
Fixed in version 4.1.3

**Comment by CervDotBe** [comment]
> Fixed in version 4.1.3
Thank you pippyn. This is like a super-super fast hotfix!
